config/initializers/rack_mini_profiler.rb

```diff
@@ -0,0 +1,7 @@
+if defined? Rack::MiniProfiler
+  # enable profiling for all pages; per-default it is disabled on production-env
+  Rack::MiniProfiler.config.authorization_mode = :allow_all
```

Comment:
Can't this be made a setting? If you intend this to be used in production.

Reply:
I guess it can, but the plan is to add the `bundler.d/profiling.rb` in a separate rpm-package so it is enabled, when that one is installed and automatically disabled, if the package is removed (which removes the bundler-file ).

An additional setting would make this more complicated.

Reply:
That depends on the default value for that setting. You can use `SETTINGS.fetch(:profiler_authorization_mode, :allow_all)` for example. Then you can add it to `settings.yaml` override it, but still have the same default.

Reply:
@ekohl do I have to document the new `SETTINGS`-setting anywhere?

Reply:
You can include it in the example settings file. I beleve that's the best place